pyspark_ai/prompt.py

```diff
@@ -40,13 +40,85 @@
     template=SQL_TEMPLATE,
 )
 
-SPARK_SQL_EXAMPLES = [
+SPARK_SQL_EXAMPLES_NO_VECTOR_SEARCH = [
     """QUESTION: Given a Spark temp view `spark_ai_temp_view_14kjd0` with the following sample vals,
     in the format (column_name: type, [sample_value_1, sample_value_2...]):
 ```
-(a: STRING, [Kongur Tagh, Grossglockner])
-(b: INT, [7649, 3798])
-(c: STRING, [China, Austria])
+(a: string, [Kongur Tagh, Grossglockner])
+(b: int, [7649, 3798])
+(c: string, [China, Austria])
+```
+Write a Spark SQL query to retrieve from view `spark_ai_temp_view_14kjd0`: Find the mountain located in Japan.
+Thought: The column names are non-descriptive, but from the sample values I see that column `a` contains mountains
+and column `c` contains countries. So, I will filter on column `c` for 'Japan' and column `a` for the mountain.
+I will use = rather than "like" in my SQL query because I need an exact match.
+Action: query_validation
+Action Input: SELECT `a` FROM `spark_ai_temp_view_14kjd0` WHERE `c` = 'Japan'
+Observation: OK
+Thought:I now know the final answer.
+Final Answer: SELECT `a` FROM `spark_ai_temp_view_14kjd0` WHERE `c` = 'Japan'"""
+    """QUESTION: Given a Spark temp view `spark_ai_temp_view_93bcf0` with the following columns:
+```
+Product: string
```

Comment:
we should unify the output here even when there area no sample values, comparing to 
```
(a: string, [Kongur Tagh, Grossglockner])
(b: int, [7649, 3798])
(c: string, [China, Austria])
```
But we can do it in a follow-up PR